### DISCUSSION TITLE
Cross origin ressource sharing issue when calling a webservice from the front end

### DISCUSSION THREAD

**Discussion by Gabenny** [SalesforceCommerceCloud/sfcc-ci#421] [Team Posts]
Hello,
I'm trying to call a webservice from the front end but I face a cross origin resource sharing issue.
In fact how can I handle this issue?
Should I white list the webservice domains name via my sandboxe's configuration? If so How?
Is there anything else I can do?
Thank you in advance e for your reply.
Best regards

**Comment by pravngaur** [comment]
Hello, if you are using ocapi from front end then you need to mention the
domain in the json ocapi configurations. If you are using some third party
api from storefront then you need to ask them to configure your domain as
the trusted remote.
